ports/esp8266/Makefile

```diff
@@ -165,23 +165,36 @@ $(BUILD)/uart.o: $(CONFVARS_FILE)
 
 FROZEN_EXTRA_DEPS = $(CONFVARS_FILE)
 
+#MAKE_PINS = boards/make-pins.py
+#BOARD_PINS = boards/$(BOARD)/pins.csv
+#AF_FILE = boards/stm32f4xx_af.csv
+#PREFIX_FILE = boards/stm32f4xx_prefix.c
+#GEN_PINS_SRC = $(BUILD)/pins_$(BOARD).c
+#GEN_PINS_HDR = $(HEADER_BUILD)/pins.h
+#GEN_PINS_QSTR = $(BUILD)/pins_qstr.h
+#GEN_PINS_AF_CONST = $(HEADER_BUILD)/pins_af_const.h
+#GEN_PINS_AF_PY = $(BUILD)/pins_af.py
+MAKE_IMG_SRC = $(CURDIR)/makeimg.py
+MAKE_IMG = $(PYTHON) $(MAKE_IMG_SRC)
```

Comment:
I don't think it's useful to have separate variables for `MAKE_IMG_SRC` and `MAKE_IMG`.  I think just have it as:
```
MAKE_IMG = $(PYTHON) $(CURDIR)/makeimg.py
```
That's how it's done in py/mkenv.py.  If a user wants to override the .py file they can just use `MAKE_IMG = $(PYTHON) mycustomscript.py`.

Also, do you need `$(CURDIR)`?